src/analytics/InjectTraits.ts

```diff
@@ -0,0 +1,28 @@
+import { PlatformPlugin, PluginType, SegmentEvent } from '@segment/analytics-react-native'
+
+/**
+ * Plugin that injects the user traits to every event
+ *
+ * Added to migrate to Segment v2 without breaking existing dashboards that rely on context_traits_*. All this data is
+ *  available as "super" properties, so we can eventually get rid of it.
+ *
+ * TODO eventually: remove dependencies and delete this plugin, see https://linear.app/valora/issue/ENG-60/remove-dependencies-on-context-traits-
+ */
+export class InjectTraits extends PlatformPlugin {
+  type = PluginType.before
+
+  execute(event: SegmentEvent) {
```

Comment:
it might be worth putting the implementation of this in a try/catch and just logging an error if it doesn't work. probably better to get some data than no data and potentially block the user

Reply:
done